### PR TITLE
feature/kakao-map : Add KakaoMap component

### PR DESCRIPTION
**Pull request by swa07016**
- Add KakaoMap component.

### CODE REVIEW

File: src/KakaoMap.js [hunk 1]
```diff
@@ -13,11 +31,114 @@ class KakaoMap extends React.Component {
...
+                    data.details ? this.createCustomOverlay(map, marker, data) : '';
+                });
+
+            });
+        };
+    };
+      
+    componentDidMount() {
+        this.mapScript();
+    }
```
Comment (soaple) [inline src/KakaoMap.js]: I think it would be better if the component lifecycle methods came after the constructor.

File: src/KakaoMap.js [hunk 2]
```diff
@@ -13,11 +31,114 @@ class KakaoMap extends React.Component {
...
+
+        kakao.maps.event.addListener(
+            marker,
+            'click',
+            this.createMouseClickListener(map, customOverlay)
+        );
+    }
+
+    mapScript = () => { 
+
```
Comment (soaple) [inline src/KakaoMap.js]: Please remove this empty line.

File: src/KakaoMap.js [hunk 3]
```diff
@@ -1,10 +1,28 @@
...
+`;
+
+const Content = styled.div`
+    padding:6px;
+    margin: 4px;
+    box-shadow: 0px 0px 5px 0px rgba(0,0,0,0.75);
+    border-radius: 16px;
+    background-color: ${(props) => props.iwBackgroundColor || "#fff"};
+    color: ${(props) => props.iwFontColor || "#000"};
+    font-size: ${(props) => props.iwFontSize || "14px"};
```
Comment (soaple) [inline src/KakaoMap.js:20]: What is the prefix 'iw' means?

**Review by soaple — CHANGES_REQUESTED** [review]
Please check the commens.

File: src/KakaoMap.js (hunk 3 above)
Comment (swa07016) [reply src/KakaoMap.js:20]: 'iw' means infowindow.

**Comment by swa07016** [comment]
I moved the componentDidmount and removed empty lines

**Review by soaple — APPROVED** [review]
LGTM (Looks Good To Me)

**Pull request merged**